Makefile

```diff
@@ -1,9 +1,10 @@
-dev:
+dev-build:
 	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
+
+dev: dev-build
```

Comment:
this tells make, if you're going to do this step then `dev-build` is a dependency of this step.